terra-jupyter-python/tests/smoke_test.py

```diff
@@ -25,31 +29,30 @@ def test_pandas():
     }
   )
   
-def test_pandas_gbq():
-  import pandas_gbq
+# def test_pandas_gbq():
```

Comment:
Instead of commenting out, could you you mark this test as skipped with a decorator like this?
`@pytest.mark.skip(reason="issues passing in credentials, see IA-4061")`